packages/components/src/DateTimePickers/LegacyDateTimePickers/DateTime/Validation/__snapshots__/Validation.component.test.js.snap

```diff
@@ -67,7 +67,6 @@ exports[`DateTime.Validation should render 1`] = `
       aria-label="Validate the date picker value"
       className="btn btn-primary"
       name="action-datepicker-validate"
-      role="button"
```

Comment:
This was removed because it's the implicit role of a button. Snapshot is impacted